.github/CODEOWNERS

```diff
@@ -6,13 +6,13 @@
 # Unless a later match takes precedence, they will be requested for review when someone opens a pull request.
 * @grafana/helm-charts-admins
 
-/charts/grafana/ @maorfr @rtluckie @torstenwalter @Xtigyro @zanhsieh
+/charts/grafana/ @maorfr @torstenwalter @Xtigyro @zanhsieh
```

Comment:
shouldn't have any code owner changes in this PR

Reply:
I think this was the result of merging in grafana:main into my fork. I'm not seeing this change in the list of diffs any longer though.